include/seastar/http/exception.hh

```diff
@@ -163,3 +163,12 @@ SEASTAR_MODULE_EXPORT_END
 }
 
 }
+
+SEASTAR_MODULE_EXPORT
+template <>
+struct fmt::formatter<seastar::httpd::base_exception> {
+    constexpr auto parse(format_parse_context& ctx) { return ctx.begin(); }
+    auto format(const seastar::httpd::base_exception& e, fmt::format_context& ctx) const {
+        return fmt::format_to(ctx.out(), "{} ({})", e.what(), e.status());
```

Comment:
Aren't std::exception-s formattable by default with the e.what() printing?

Reply:
it does not print the `e.status()`, that's what i am missing here.